src/CPUSolver.cpp

```diff
@@ -821,8 +905,10 @@ void CPUSolver::packBuffers(std::vector<long> &packing_indexes) {
   for (int i=0; i < num_domains; i++) {
     int send_domain = _neighbor_domains.at(i);
 
-    /* Reset send buffers */
-    int start_idx = _fluxes_per_track + 1;
+    /* Reset send buffers : start at beginning if the buffer has not been
+       prefilled, else start after what has been prefilled */
+    int start_idx = (_send_buffers_index.at(i)) * (_track_message_size) +
```

Comment:
two pairs of brackets are useless here. remove them.

Reply:
Done